warehouse/models/mart/gtfs/_mart_gtfs.yml

```diff
@@ -1304,3 +1358,62 @@ models:
+            - feed_key
+      - dbt_utils.expression_is_true:
+          expression: "ttl_service_hours >= 0"
+    columns:
+      - name: service_date
+        description: Date on which this service level was present
+        tests:
+          - not_null
+      - name: feed_key
+        tests:
+          - not_null
+        description: feed_key only present with active service on date
+      - name: ttl_service_hours
+        description: Total service hours across entire feed on date
+      - name: n_trips
+        description: Total number of trips ran on date, if any
+      - name: first_departure_ts
+        description: Time of first departure on date
+      - name: last_arrival_ts
+        description: Time of last departure on date
+      - name: n_stop_times
+        description: |
+          Total number of stop events observed
+      - name: n_routes
+        description: |
+          Total number of routes that had trips scheduled on this day
+      - name: contains_warning_duplicate_stop_times_primary_key
+        description: |
+          Rows with `true` in this column indicate that the columns in this table that are aggregated from
+          stop times data (`n_stops`, `n_stop_times`,  `trip_first_departure_ts`, `trip_last_arrival_ts`,
+          and `service_hours`) contain at least one row that had a duplicate primary key in the source stop times data.
+
+          I.e., at least one row being aggregated had a `trip_id` / `stop_sequence` pair that was not unique
+          in the input data. This indicates that data quality issues were present in the stop times data
+          that is being summarized here, and counts may be inflated due to multiple rows with identical identifiers.
+      - name: contains_warning_duplicate_trip_primary_key
+        description: |
+          Rows with `true` in this column indicate that `dim_trips` contains duplicates of this trip primary key.
+
+          i.e., `trip_id` is duplicated within an individual feed instance.
+      - name: contains_warning_missing_foreign_key_stop_id
+        description: |
+          # Like I mentioned in the table itself, if we're not carrying over n_stops do we need to keep this?
+          # Another option is to keep the field and then just delete the second half of this description?
+
+          Rows with `true` in this column indicate that the columns in this table that are aggregated from
+          stop times data (`n_stops`, `n_stop_times`,  `trip_first_departure_ts`, `trip_last_arrival_ts`,
+          and `service_hours`) contain at least one row that had a missing `stop_id` foreign key in the source stops data.
+
+          #I.e., at least one row being aggregated had a `stop_id` foreign key that was missing
+          #in the input data. This indicates that data quality issues were present in the stop times data
+          #that is being summarized here, and the count of distinct `stop_id`s at the trip level may be incorrect.
```

Comment:
remove the last part of this sentence, but keep up to `data quality issues were present in the stop times data that is being summarized here`.